spec/src/main/asciidoc/rest-endpoints.adoc

```diff
@@ -345,239 +345,147 @@ then `OPTIONS /metrics/base` exposes:
 
-Families and OpenMetrics base units are:
 
+.Prometheus format mapping for a Histogram metric
+[cols="6,4,8,3"]
 |===
-| Family | Base unit
+| Suffix{label}                   | TYPE    | Value (Histogram method)            | Units
 
-| Bits    | bytes
-| Bytes   | bytes
-| Time   | seconds
-| Percent | ratio (normally ratio is A_per_B, but there are exceptions like `disk_usage_ratio`)
+| `<units>_max`                   | Gauge   | `getSnapshot().getMax()`            | <units>
+| `<units>{quantile="0.5"}`       | Summary | `getSnapshot().getMedian()`         | <units>
+| `<units>{quantile="0.75"}`      | Summary | `getSnapshot().get75thPercentile()` | <units>
+| `<units>{quantile="0.95"}`      | Summary | `getSnapshot().get95thPercentile()` | <units>
+| `<units>{quantile="0.98"}`      | Summary | `getSnapshot().get98thPercentile()` | <units>
+| `<units>{quantile="0.99"}`      | Summary | `getSnapshot().get99thPercentile()` | <units>
+| `<units>{quantile="0.999"}`     | Summary | `getSnapshot().get999thPercentile()`| <units>
+| `<units>_count`                 | Summary | `getCount()`                        | <units>
+| `<units>_sum`                   | Summary | `getSum()`                          | <units>
 |===
 
-*Counters*
-
-`Counter` metrics are considered dimensionless. The implementation must not append the unit name to the metric name and must not scale the value.
-
-
-*Meters and Timers*
-
-`Meter` and `Timer` have fixed units as described below regardless of the `unit` value in the Metadata.
 
-==== Gauge OpenMetrics Text Format
 
-The value of the gauge must be the value of `getValue()` with appropriate naming/scaling based on  <<OpenMetrics_units>>
-The OpenMetrics name is composed `<scope>'_'<metric-name>'_'[<unit>'_']`.
+==== Timer
 
-.Example OpenMetrics text format for a Gauge in dollars.
+.Example Timer in Prometheus format.  Timers use `seconds` as the unit.
 [source, ruby]
 ----
-# TYPE application_cost_dollars gauge
-# HELP application_cost_dollars The running cost of the server in dollars.
-application_cost_dollars 80
+# HELP myClass_myMethod_seconds duration of myMethod <1>
+# TYPE myClass_myMethod_seconds summary <2>
+myClass_myMethod_seconds{scope="vendor",quantile="0.5"} 0.0524288 <3>
+myClass_myMethod_seconds{scope="vendor",quantile="0.75"} 0.0524288 <3>
+myClass_myMethod_seconds{scope="vendor",quantile="0.95"} 0.054525952 <3>
+myClass_myMethod_seconds{scope="vendor",quantile="0.98"} 0.054525952 <3>
+myClass_myMethod_seconds{scope="vendor",quantile="0.99"} 0.054525952 <3>
+myClass_myMethod_seconds{scope="vendor",quantile="0.999"} 0.054525952 <3>
+myClass_myMethod_seconds_count{scope="vendor"} 100.0 <3>
+myClass_myMethod_seconds_sum{scope="vendor"} 5.310349419 <3>
+# HELP myClass_myMethod_seconds_max duration of myMethod <1>
+# TYPE myClass_myMethod_seconds_max gauge <2>
+myClass_myMethod_seconds_max{scope="vendor"} 0.05507899 <3>
```

Comment:
Do we still plan to use an underscore prefix for built-in scopes? If so the example should reflect that.

Reply:
yes, fixed!

Reply:
You actually addressed something I missed!

I was remembering that the _values_ for the built-in scopes--`application`, `vendor`, and `base`--were going to be prefixed with underscore. Or did I misremember that?

Reply:
I don't remember us talking about putting an underscore in front of the values (just the scope label).  Since it's our own (new) label I wouldn't think we'd need to worry about collisions with the values.

Reply:
Yes, I couldn't recall all the details of where we came out on that. 

I guess adding an underscore to the _value_ might be overdoing it and might actually be counterproductive. We would burden with yet another migration hurdle any users who already have procedures (or simply expectations) of the names from earlier MP metrics releases as `application`, `vendor`, and `base`.